<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>P_PageNavigator</class>
 <widget class="QWidget" name="P_PageNavigator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>573</width>
    <height>30</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
/* 选项卡中的 按钮 样式 */
QToolButton{
	font: 75 10pt &quot;黑体&quot;;
	font-weight:bold;
	background:  rgba(255, 255, 255,0);
}
QToolButton:hover{
	color:#ffffff;
	border-radius:6px;
	border:1px solid #ffffff;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #8ad8ff, stop:0.505 #5fc2ff, stop:1 #5fc2ff);
}
QToolButton:checked{
	color:#ffffff;
	border-radius:6px;
	border:1px solid #ffffff;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0.2 #5075BC, stop:0.305 #557CC7, stop:1 #5C87D9);
}

QLabel{
	font: 75 10pt &quot;黑体&quot;;
}

</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1,0">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QToolButton" name="toolButton_last">
       <property name="minimumSize">
        <size>
         <width>28</width>
         <height>28</height>
        </size>
       </property>
       <property name="text">
        <string>&lt;&lt;</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/code_piece_qt/Resources/images/global/drill_up.png</normaloff>:/code_piece_qt/Resources/images/global/drill_up.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextOnly</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_start">
       <property name="minimumSize">
        <size>
         <width>28</width>
         <height>28</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>28</height>
        </size>
       </property>
       <property name="text">
        <string>1</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/code_piece_qt/Resources/images/button/按钮-搜索.png</normaloff>:/code_piece_qt/Resources/images/button/按钮-搜索.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextOnly</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_left">
       <property name="maximumSize">
        <size>
         <width>16</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>…</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_centerBtn"/>
     </item>
     <item>
      <widget class="QLabel" name="label_right">
       <property name="maximumSize">
        <size>
         <width>16</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>…</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_end">
       <property name="minimumSize">
        <size>
         <width>28</width>
         <height>28</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>28</height>
        </size>
       </property>
       <property name="text">
        <string>99</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/code_piece_qt/Resources/images/button/按钮-搜索.png</normaloff>:/code_piece_qt/Resources/images/button/按钮-搜索.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextOnly</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_next">
       <property name="minimumSize">
        <size>
         <width>28</width>
         <height>28</height>
        </size>
       </property>
       <property name="text">
        <string>&gt;&gt;</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/code_piece_qt/Resources/images/button/按钮-搜索.png</normaloff>:/code_piece_qt/Resources/images/button/按钮-搜索.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextOnly</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="label_1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>第</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_jumpTo">
       <property name="minimumSize">
        <size>
         <width>28</width>
         <height>28</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>页</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_jumpTo">
       <property name="minimumSize">
        <size>
         <width>54</width>
         <height>28</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>54</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>跳转</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/code_piece_qt/Resources/images/button/按钮-搜索.png</normaloff>:/code_piece_qt/Resources/images/button/按钮-搜索.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextOnly</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../code_piece_qtutil.qrc"/>
 </resources>
 <connections/>
</ui>
